network/network.go

```diff
@@ -89,6 +90,11 @@ type DNSInfo struct {
 	Options []string
 }
 
+func (nwInfo *NetworkInfo) String() string {
```

Comment:
In NPM, we elected to use a method name different than `String()` to make investigations easier. If you print the NetworkInfo pointer or print it with "%v", then it will implicitly use the `String()` method, which might not have all the info in the struct.

https://github.com/Azure/azure-container-networking/pull/1235

Stringer interface:
https://towardsdev.com/type-stringer-golang-88d89777b959